.github/CHANGELOG.md

```diff
@@ -1,105 +1,188 @@
   `clique` and `subgraph` modules.
-  [#295](https://github.com/XanaduAI/strawberryfields/pull/295)
-  [#296](https://github.com/XanaduAI/strawberryfields/pull/296)
-  [#297](https://github.com/XanaduAI/strawberryfields/pull/297)
-
-### Improvements
-* Added two-mode squeezed operation support as a primitive, rather than simply
-  through decomposition, using The Walrus for fast computation.
-  [#289](https://github.com/XanaduAI/strawberryfields/pull/289)
-
-* Added The Walrus implementations for the displacement, squeezing and beamsplitter 
-  operations to improve speed.
-  [#287](https://github.com/XanaduAI/strawberryfields/pull/287)
-  
-* Added custom tensor contractions for the beamsplitter and the two-mode squeeze
-  gate as well as faster application of diagonal gate matrices.
-  [#292](https://github.com/XanaduAI/strawberryfields/pull/292)
-  
-* Moved apply-gate functions to `Circuit` class, and removed `apply_gate_einsum` and
-  `Circuits._apply_gate`, since they were no longer used.
-  [#293](https://github.com/XanaduAI/strawberryfields/pull/293/)
-  
-* Unified backend results returned from running simulators and added checks for
-  using batching, post-selection and feed-fowarding together with multiple
-  shots, which now raises an error.
-  [#300](https://github.com/XanaduAI/strawberryfields/pull/300)
-  
-* Modified the decomposition to ensure that identity-like unitaries are implemented 
-  with no swaps.
+  [(#295)](https://github.com/XanaduAI/strawberryfields/pull/295)
+  [(#296)](https://github.com/XanaduAI/strawberryfields/pull/296)
+  [(#297)](https://github.com/XanaduAI/strawberryfields/pull/297)
+
+<h3>Improvements</h3>
+
+* Moved Fock backend apply-gate functions to `Circuit` class, and removed
+  `apply_gate_einsum` and `Circuits._apply_gate`, since they were no longer used.
+  [(#293)](https://github.com/XanaduAI/strawberryfields/pull/293/)
+
+* Results returned from all backends now have a unified type and shape.
+  In addition, attempting to use batching, post-selection and feed-foward together
+  with multiple shots now raises an error.
+  [(#300)](https://github.com/XanaduAI/strawberryfields/pull/300)
+
+* Modified the rectangular decomposition to ensure that identity-like
+  unitaries are implemented with no swaps.
   [#311](https://github.com/XanaduAI/strawberryfields/pull/311)
 
-### Bug fixes
+<h3>Bug fixes</h3>
 
 * Symbolic Operation parameters are now compatible with TensorFlow 2.0 objects.
-  [#282](https://github.com/XanaduAI/strawberryfields/pull/282)
+  [(#282)](https://github.com/XanaduAI/strawberryfields/pull/282)
 
 * Added `sympy>=1.5` to the list of dependencies.
   Removed the `sympy.functions.atan2` workaround now that SymPy has been fixed.
-  [#280](https://github.com/XanaduAI/strawberryfields/pull/280)
-  
+  [(#280)](https://github.com/XanaduAI/strawberryfields/pull/280)
+
 * Removed two unnecessary else statements that pylint complained about.
```

Comment:
```suggestion
* Removed two unnecessary else statements.
```
imo, not even worth putting in changelog

Reply:
agree